README.md

```diff
@@ -59,6 +59,24 @@ Other than requiring Python3.x , you require the following libraries:<br>
    A file called 'Table.html' would be made in the folder itself.
    You can open it and see the table.
    
```

Comment:
If you want these to be merged in `master`, you also need to add the latest `graph.py` file.
Also, changes in the README.md should be made clarifying these 2 paths to run the program,